src/lib.rs

```diff
@@ -99,7 +99,10 @@ pub use lazy_static;
 pub use query::Fetch;
 
 #[cfg(feature = "macros")]
-pub use hecs_macros::{Bundle, Query};
+#[doc(hidden)]
+pub use hecs_macros::DynamicClone;
```

Comment:
`DynamicClone` is defined in this crate, not `hecs_macros`, and the reexport should be part of the block that is commented with `// Unstable implementation details needed by the macros`.

Reply:
Alright